Formula/trust-dns.rb

```diff
@@ -0,0 +1,47 @@
+class TrustDns < Formula
+  desc "Rust based DNS client, server, and Resolver"
+  homepage "https://github.com/bluejekyll/trust-dns"
+  url "https://github.com/bluejekyll/trust-dns/archive/refs/tags/v0.20.3.tar.gz"
+  sha256 "1766f59ea28e1c1289fcd370d455ae73416814035bad1de313528391cbf8454a"
+  license "MIT"
+
```

Comment:
```suggestion

  livecheck do
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/i)
  end

```

By default, livecheck checks the Git tags but it gives `0.21.0-alpha.2` as newest instead of `0.20.3`. livecheck automatically omits certain prerelease tags based on suffix (e.g., `-alpha`) but it doesn't handle a suffix like `-alpha.2`, so we need to restrict matching to the stable tags using a regex. The standard regex for Git tags like `1.2.3`/`v1.2.3` (as in the suggestion) is appropriate in this context.